packages/nextjs/next.config.js

```diff
@@ -9,6 +9,7 @@ const nextConfig = {
   eslint: {
     ignoreDuringBuilds: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true",
   },
+  transpilePackages: ["se-2"],
```

Comment:
could you explain please, why we need it?

Reply:
if you look at https://github.com/scaffold-eth/se-2/pull/231#issuecomment-1470438008 in the section 

> Approaches explored :
> 2. 

TLDR;
 Since we have `scaffold.config.ts` outside the `nextjs` directory if you don't pass in this it gives this error -> https://github.com/scaffold-eth/se-2/pull/231#discussion_r1135317725
 
 The reason I have put `se-2` there is because we had `scaffold.config.ts` at `se-2` workspace and needed to transpile it during `next` build  : 
 
![Screenshot 2023-03-16 at 9 16 32 PM](https://user-images.githubusercontent.com/80153681/225674004-8798ef4a-a4de-408b-ada4-65525a1a957e.jpg)

Although would highly recommend giving a quick read of the section  `Approaches explored ` especially the links  mentioned in that section 🙌

Also would love to hear if there are any alternatives / suggestions 🙌